wormhole-connect/src/views/v2/Bridge/AssetPicker/TokenList.tsx

```diff
@@ -74,6 +74,24 @@ const TokenList = (props: Props) => {
     const tokens: Array<TokenConfig> = [];
 
     const addToken = (tokenConfig: TokenConfig) => {
+      // Exclude frankenstein tokens with no balance
+      const balance = Number(balances?.[tokenConfig.key]?.balance);
+      if (
+        isFrankensteinToken(tokenConfig, selectedChainConfig.key) &&
+        !balance
+      ) {
+        return;
+      }
+
+      // Exclude wormhole-wrapped tokens with no balance
+      if (
+        props.isSource &&
```

Comment:
is this backwards? did you mean to hide them in the destination list? the source list should already not show tokens without a balance.

Reply:
No this is right, we want to be able to select wormhole-wrapped tokens on the destination chain without a balance (otherwise we can't transfer them). We just disable these tokens when they have 0 balance (but they're still shown). This makes it so that they're not even shown

Reply:
got it